sdks/python/apache_beam/transforms/core.py

```diff
@@ -882,6 +882,9 @@ def make_fn(self, fn):
   def default_label(self):
     return '%s(%s)' % (self.__class__.__name__, self._fn_label)
 
+  def process_argspec_fn(self):
+    return self.fn._fn  # pylint: disable=protected-access
```

Comment:
Thank you. This change already went in, I will fix this in a follow up.

 `CombineFn` does not expose process function similar to `DoFn`. What should it do in the default case, (base implementation for `pocess_argspec_fn`)?